### PR TITLE
[IMP] account, l10n_cl: dashboard style

### PR DESCRIPTION
**Pull request by h4818**
- Reduce the space between the left buttons and summary on the right of dashboard kanban cards
- fix typo
- Add link to unreconciled payments (moved from account_reports)
- remove unused file (account_dashboard_setup_bar.xml)
- remove New Transaction links/buttons
- add a bank account in demo data if there isn't one and link it to the bank journal
- hide setup bank if there is a bank account on the journal
- link btn style: color
- Simplify kanban summary text: X ~~Invoices~~ to Validate, Y ~~Bills~~ to Pay
- Simplify buttons: New ~~Invoice/Bill/Entry~~
- Remove hover tooltips on sample data
- FIX: when calling `create_document_from_attachment` in JS, the help markup is lost. Ensure that it is still markup.

journal form:
- display the statement source field in invoicing enterprise

To reproduce the no content helper issue:
- drag and drop a file to the bank journal
- on redirection to the bank recon kanban, search for "somethingarbitrary"
- see HTML text as the helper

why change things in l10n_cl?

TLDR; Chile demo data was forcing the delay of writing fields in demo data records. By making changes to the demo data, we can improve the demo data load process.

When a COA does not have a bank account in demo data, a new res_partner_bank is created. A partner is required for this bank, but since Chile had it's own partner demo data, the field partner_id is delayed causing a NOT NULL constraint error when creating the partner bank.
This uncovered a few issues:
1. in the demo data loading mechanism in `account`: a relational field with an id (integer) value, should not be delayed. The record obviously exists.
2. an empty dict in demo data should not be considered as a data model to be created.
3. l10n_cl: demo data has unnecessary delays due to the updates of partner taxpayer types & latam document types. Demo partners can be updated in XML by rearranging the sequence of the demo files & the document type was already active.
4. l10n_cl: by avoiding the delays, we can avoid the invalid computation of the account move's latam document type.

The invalid computation problem (point 4 above) still needs to be resolved...ie. foreign partners should use a different set of codes: `_compute_l10n_latam_document_type` and `_check_document_types_post` are not aligned.

### CODE REVIEW

**Review by clbr-odoo — APPROVED** [review]
LGTM, is it tested by a PO ?

Just FYI, few inconsistencies I noticed testing the UI:
- for purchases we have "New", while for sales and misc we still have "New Invoice/Entry"
- for purchases we have "Upload" in primary, "New" in secondary, it's the opposite for Misc.

**Comment by h4818** [comment]
> LGTM, is it tested by a PO ?
> 
> Just FYI, few inconsistencies I noticed testing the UI:
> 
> * for purchases we have "New", while for sales and misc we still have "New Invoice/Entry"
> * for purchases we have "Upload" in primary, "New" in secondary, it's the opposite for Misc.

Well spotted.
- for "new invoice" I think its ok, since there isn't a secondary button, "new entry" could maybe change to new
- as for primary/secondary, that's correct. You're more likely to upload a bill, but manually capture an entry

@chklop

**Comment by chklop** [comment]
New is probably better everywhere, it's also the title of the button everywhere (list view, form view). Everyone knows what "New" is going to do, as it's a button contextualized to a journal.

File: addons/account/views/account_dashboard_setup_bar.xml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-<odoo>
-    <record id="account_journal_dashboard_kanban_view_js" model="ir.ui.view">
```
Comment (clbr-odoo) [inline addons/account/views/account_dashboard_setup_bar.xml:2]: Forgot to say I think you'll need to add a fix upgrade script to remove this one. :slightly_smiling_face:

Comment (h4818) [reply addons/account/views/account_dashboard_setup_bar.xml:2]: nope, not required.
the file was not in the manifest (removed in 12.0 :D ), and also, it's an inherited view...

Comment (h4818) [reply addons/account/views/account_dashboard_setup_bar.xml:2]: https://github.com/odoo/upgrade/pull/6145#issuecomment-2161186036

**Comment by h4818** [comment]
@odoo/rd-security Python is already returning a Marked up no content helper, but it somehow get's lost in translation with the JS action. The markup is static, and overridden in the corresponding enterprise [PR](https://github.com/odoo/enterprise/pull/63933/files#diff-b2eee9dea3fac397aa4007862cfe98a18ce9048cd4bbe61dc13da36ffaa18d03R20-R28) which needs an override too

https://github.com/odoo/enterprise/blob/1efff07237abac109afca94f6a2579ae74579242/account_bank_statement_import/models/account_journal.py#L24-L30

https://github.com/odoo/enterprise/blob/1efff07237abac109afca94f6a2579ae74579242/account_bank_statement_import/models/account_journal.py#L85-L92

https://github.com/odoo/enterprise/blob/cd5ea9e6ff89f4b63e8c60f7c8a26e321f7ca1e3/account_accountant/models/account_bank_statement.py#L67-L70

**Review by clbr-odoo — APPROVED** [review]
@robodoo r+

**Comment by clbr-odoo** [comment]
@robodoo r+
